azure_monitor/README.rst

```diff
@@ -0,0 +1,17 @@
+Azure Monitor Exporters using OpenTelemetry
+===========================================
+
+This library provides integration with Azure Monitor using OpenTelemetry.
+
+Installation
+------------
+
+::
+
+    pip install azure-monitor-opentelemetry-exporters-python
```

Comment:
This name is very weird.

Reply:
Renamed to `opentelemetry-azure-monitor-exporter`